auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -32,12 +32,15 @@ class CheckPullRequest extends RequestHandler {
     final GithubService gitHub = await config.createGithubService();
     final _AutoMergeQueryResult queryResult = await _parseQueryData(pullRequest, gitHub);
     if (await shouldMergePullRequest(queryResult, slug, gitHub)) {
-      // TODO(Kristin): Keep pulling pubsub queue, https://github.com/flutter/flutter/issues/98704
-
+      log.info('Merge the pull request: ${queryResult.number}');
+      await gitHub.merge(slug, queryResult.number);
```

Comment:
Do we want to hold on this logic until we validate all logics are correct? Additionally the app is now only in dev repo, and it does not make sense to merge the PR there.

Reply:
Gotcha, I will make this a TODO at this time.